infra/cmake/packages/HDF5SourceConfig.cmake

```diff
@@ -0,0 +1,18 @@
+function(_HDF5Source_import)
+  if(NOT DOWNLOAD_HDF5)
+    set(HDF5Source_FOUND FALSE PARENT_SCOPE)
+    return()
+  endif(NOT DOWNLOAD_HDF5)
+
+  nnas_include(ExternalSourceTools)
+  nnas_include(OptionTools)
+
+  envoption(HDF5_URL https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.8/hdf5-1.8.16/src/hdf5-1.8.16.tar.gz)
```

Comment:
Can you check this URL can be downloaded in our in-house git CI ?

Reply:
Ummm.. I can't come up with how to do it.. @hseok-oh Could you let me know how to do this? There is no test space in nnfw_internal jenkins.

Reply:
I'll change above URL to github one that can be used in CI.